src/ShallowTraversal.js

```diff
@@ -29,7 +29,7 @@ export function childrenOfNode(node) {
 
 export function hasClassName(node, className) {
   const classes = propsOfNode(node).className || '';
-  return ` ${classes} `.indexOf(` ${className} `) > -1;
+  return ` ${classes.trim()} `.indexOf(` ${className} `) > -1;
```

Comment:
It's a class name - the browser will always collapse whitespace into a single space anyways, I thought.

I totally agree though that what `document.querySelector` does should be our guide here - but i was under the impression that the array of class names on an element was as if you'd done `.split(/\s+/g)` on it.